.eslintignore

```diff
@@ -1 +1 @@
-tests/*
+tests
```

Comment:
Do we any to remove lint from tests. I removed this from the original and it highlighted a few issues. With `jest: true` in the eslint config, it should work fine or is there another reason why you've disabled it?

Reply:
Yeah I found that Husky complains with listing errors when you try to commit. Does it not do that for you?

Reply:
Just gave it another try, works fine for me. Runs `eslint --fix` and passes... wonder if it's not picking up the eslint config.

![husky-serverless-bundle](https://user-images.githubusercontent.com/10468110/87348505-c0bcb380-c54c-11ea-8b8c-be0f676d3786.gif)

Reply:
Ah sorry for the confusion. Yeah previously I didn't want linting for the tests. But if you've fixed all the tests. Then let's stick with it.